modules/canbus/vehicle/gem/gem_controller.cc

```diff
@@ -422,9 +421,9 @@ void GemController::Steer(double angle, double angle_spd) {
     AINFO << "The current driving mode does not need to set steer.";
     return;
   }
-  const double real_angle = params_.max_steer_angle() * angle / 100.0;
+  const double real_angle = params_.max_steer_angle() * angle / 100.0 * 3.1415/180.0;
 
-  const double real_angle_spd = (
+  const double real_angle_spd = 3.1415 / 180.0 * (
```

Comment:
double check this statement.  Is it equivalent to M_PI / 180.0 * params_max_steer_angle_spd() * angle_spd / 100.0?

Reply:
@startcode Will just cherry-pick the commit as is and do the cleaning afterwards, before merging the branch back to master.